Clinical-Genomics/scout
# Add more consequences to SO_TERMS

This PR adds a functionality or fixes a bug.

There are consequences (SO terms) [listed in the VEP documentation](https://www.ensembl.org/info/genome/variation/prediction/predicted_data.html) that are not in `SO_TERMS`. This pull request adds them. The severity seems to match with what VEP lists, so I just added these accordingly.

**How to test**:
1. Try to load a case with that has variants whose consequence is either `splice_donor_5th_base_variant`, `splice_donor_region_variant` or `splice_polypyrimidine_tract_variant`.
2. See the error message below.

    <details>
    <summary>Error message</summary>

    ```
    2022-08-18 16:29:03 vs842 scout.adapter.mongo.variant_loader[1966] INFO Start inserting clinical cancer variants into database
    2022-08-18 16:29:03 vs842 scout.adapter.mongo.variant_loader[1966] ERROR unexpected error
    Traceback (most recent call last):
      File "/scout_prod/scout/scout/adapter/mongo/variant_loader.py", line 701, in load_variants
        nr_inserted = self._load_variants(
      File "/scout_prod/scout/scout/adapter/mongo/variant_loader.py", line 432, in _load_variants
        parsed_variant = parse_variant(
      File "/scout_prod/scout/scout/parse/variant/variant.py", line 285, in parse_variant
        for parsed_transcript in parse_transcripts(raw_transcripts):
      File "/scout_prod/scout/scout/parse/variant/transcript.py", line 80, in parse_transcripts
        transcript["region_annotations"] = get_regional_annotation(functional_annotations)
      File "/scout_prod/scout/scout/parse/variant/transcript.py", line 292, in get_regional_annotation
        regional_list.append(SO_TERMS[annotation]["region"])
    KeyError: 'splice_polypyrimidine_tract_variant'
    ```
    </details>

3. Do `scout view cases` and see that the case is still added.

**Expected outcome**:
Variants should be loaded without errors.

**Review:**
- [x] tests executed by automatic tests.
